app/views/lists/show.html.haml

```diff
@@ -0,0 +1,19 @@
+%h5.float-center= @list.name
+- books = []
+- books_id = []
+- @books_ids.each{ |books| books_id.push(books.book_id) }
+- books_id.each do |book|
+  - books.push(Book.find(book))
+- if books.empty?
+  %h6.float-center You got nothing here yet
+.row
+  - books.each do |book|
+    .card.text-center.mx-auto.mb-3{:style => "width: 18rem;"}
+      - if book.cover.attached?
+        = link_to image_tag( book.cover, width: "250pt" ), book_path(book), class: "card-img-top"
+      - else
+        = link_to image_tag( "NoBookCover.png", width: "250pt" ), book_path(book), class: "card-img-top"
+      .card-body
+        = book.title
+        %br/
+        = link_to "Delete", { controller: 'books_lists', action: "destroy", id: book, list_id: @list }, remote: true, method: :delete
```

Comment:
and here `, method: :delete` works just fine for you :D 

dude, copy-paste carefully, I'm begging you

also `action: :destroy`, avoid string literals